src/pages/components/video-player.mdx

```diff
@@ -29,6 +30,15 @@ The Video player aspect ratio should match that of the video, or use one of the
 </Column>
 </Row>
 
+## Content guidance
+
+| Field | Field type | Required   | Cardinality | Notes / Options |
+| -------- | --------- | ---------- | ---------- | ----------|
+| Video ID | Short copy | Yes | 1 | - |
+| Caption | Short copy | No | 1 | Could be provided through video source |
+| Show caption | True/False | Yes | 1 | True = Show caption<br/>False = Don't show caption |
```

Comment:
So something isn't quite right here to me. I don't know if both of these need to exist. It's supposed to be the video's title and duration coming directly from Kaltura. I think the `show caption` (which is technically reversed and instead `hideCaption` `true` hides, and `false` shows) is only here because the caption is always present due to the nature of Kaltura. I think this is conflicting to me to present both in the way we're doing it.

Reply:
If we know for sure that the caption comes directly from Kaltura then I think we can get rid of this. If it's possible a CMS user would use a different video format or something and the caption would have to be manually created, I would recommend keeping it.
For "hideCaption", that's good to know! I saw it in Storybook as the "showCaption", but I'll change it in another commit. My thoughts on keeping this line applies to the "Caption" field being there too. What do you think?

Reply:
@mfearer94 do you have a link to where you saw it on storybook?

Reply:
Small detail, but this is also not required. It's false by defualt.

Reply:
As of right now we only support one video option, and as far as I'm aware all details come from the API. We can double-check with development, but I don't see an option to override it at the moment. I don't think we want users changing the title of a video on the front end for SEO purposes either at this time.

Reply:
@photodow all the way down in props: https://ibmdotcom-react-staging.mybluemix.net/?path=/docs/components-videoplayer--default

I guess that's another question. Is something required if it has a default and is a boolean. I've always thought yes, but I would love other thoughts on it.

Love that! That would make this a ton easier if Kaltura is the only one. Do we know who to talk to to confirm that?

Reply:
> I guess that's another question. Is something required if it has a default and is a boolean. I've always thought yes, but I would love other thoughts on it.

I would argue it's optional because you don't have to add the content to make the component work.

> Love that! That would make this a ton easier if Kaltura is the only one. Do we know who to talk to to confirm that?

Kaltura is definitely the only one supported.

Reply:
Awesome, I'm going to get rid of both of these since Kaltura is pretty much defining these fields.